prisma/schema.prisma

```diff
@@ -11,22 +11,23 @@ generator client {
 }
 
 model Transfer {
-  id           String         @id @default(auto()) @map("_id") @db.ObjectId
-  depositNonce Int            @unique
-  resource     Resource?      @relation(fields: [resourceID], references: [id])
-  resourceID   String?
-  fromDomain   Domain         @relation(name: "fromDomainRelation", fields: [fromDomainId], references: [id])
-  fromDomainId String
-  toDomain     Domain?        @relation(name: "toDomainRelation", fields: [toDomainId], references: [id])
-  toDomainId   String?
-  sender       String?
-  destination  String?
-  amount       String?
-  timestamp    Int?
-  status       TransferStatus
-  deposit      Deposit?
-  execution    Execution?
-  fee          Fee?
+  id            String         @id @default(auto()) @map("_id") @db.ObjectId
+  depositNonce  Int            @unique
+  resource      Resource?      @relation(fields: [resourceID], references: [id])
+  resourceID    String?
+  fromDomain    Domain         @relation(name: "fromDomainRelation", fields: [fromDomainId], references: [id])
+  fromDomainId  String
+  toDomain      Domain?        @relation(name: "toDomainRelation", fields: [toDomainId], references: [id])
+  toDomainId    String?
+  sender        String?
+  destination   String?
+  amount        String?
+  timestamp     Int?
+  status        TransferStatus
+  deposit       Deposit?
+  execution     Execution?
+  fee           Fee?
+  addressStatus String?
```

Comment:
As we are already doing this it might make more sense to have a account model that is the sender and then on that we set an addressStatus instead of doing everything for each transfer.
This way I think we are denormalizing the model unnecessarily